src/include/sof/audio/pipeline.h

```diff
@@ -124,6 +124,7 @@ struct pipeline_data {
 	uint32_t delay_ms;		/* between PRE_{START,RELEASE} and {START,RELEASE} */
 #if CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL
 	uint32_t kcps[CONFIG_CORE_COUNT]; /**< the max count of KCPS */
+	uint32_t kcps_acc[CONFIG_CORE_COUNT]; /**< accumulated KCPS before maximization */
```

Comment:
I'm not sure why this is needed only with llext? As far as I know previously it was working with modules which could send 0 cpc

Reply:
I'm not sure either, I think it's a bug not directly related to LLEXT, we just have identified it now